python/example_robot_data/robots_loader.py

```diff
@@ -258,21 +258,21 @@ def loadHyQ():
     return HyQLoader().robot
 
 
-class SoloLoader(RobotLoader):
+class Solo8Loader(RobotLoader):
     path = 'solo_description'
     urdf_filename = "solo.urdf"
     srdf_filename = "solo.srdf"
     ref_posture = "standing"
     free_flyer = True
 
 
-class Solo12Loader(SoloLoader):
+class Solo12Loader(Solo8Loader):
     urdf_filename = "solo12.urdf"
 
 
-def loadSolo(solo=True):
-    warnings.warn(_depr_msg('loadSolo()', 'solo'), FutureWarning, 2)
-    loader = SoloLoader if solo else Solo12Loader
+def loadSolo(solo8=True):
```

Comment:
I think this one should keep `solo=True`, because if someone has an old code base with `robot = loadSolo(solo=True)`, we want to load `solo8`and display the warning.

Reply:
Agreed, thanks for proposing this. I switched it back to `solo`.